gamestonk_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -463,23 +469,364 @@ def call_newsletter(self, other_args: List[str]):
 
         substack_view.display_newsletters(top=ns_parser.top, export=ns_parser.export)
 
+    @try_except
+    def call_tokens(self, other_args: List[str]):
+        """Process tokens command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="tokens",
+            description="""
+                Display tokens trade-able on Uniswap DEX
+                [Source: https://thegraph.com/en/]
+            """,
+        )
+
+        parser.add_argument(
+            "--skip",
+            dest="skip",
+            type=check_positive,
+            help="Number of records to skip",
+            default=0,
+        )
+
+        parser.add_argument(
+            "--limit",
+            dest="limit",
+            type=check_positive,
+            help="Number of records to display",
+            default=20,
+        )
+
+        parser.add_argument(
+            "-s",
+            "--sort",
+            dest="sortby",
+            type=str,
+            help="Sort by given column. Default: index",
+            default="index",
+            choices=[
+                "index",
+                "symbol",
+                "name",
+                "tradeVolumeUSD",
+                "totalLiquidity",
+                "txCount",
+            ],
+        )
+
+        parser.add_argument(
+            "--descend",
+            action="store_false",
+            help="Flag to sort in descending order (lowest first)",
+            dest="descend",
+            default=True,
+        )
+
+        parser.add_argument(
+            "--export",
+            choices=["csv", "json", "xlsx"],
+            default="",
+            type=str,
+            dest="export",
+            help="Export dataframe data to csv,json,xlsx file",
+        )
+
+        ns_parser = parse_known_args_and_warn(parser, other_args)
```

Comment:
You can use the third argument of this function, without needing to use the export above

Reply:
Can you explain me this, cause I don't understand :D Or maybe link me to an example ?

Reply:
Yes, sure! Have a look at this PR, see how I removed the "export" argument, and it is already incorporated in `parse_known_args_and_warn`

https://github.com/GamestonkTerminal/GamestonkTerminal/pull/926/files